src/crypto/dtls.rs

```diff
@@ -41,59 +42,84 @@ pub struct DtlsCert(DtlsCertInner);
 enum DtlsCertInner {
     #[cfg(feature = "openssl")]
     OpenSsl(super::ossl::OsslDtlsCert),
+    #[cfg(not(feature = "openssl"))]
+    OpenSsl(DummyCert),
```

Comment:
Wouldn't it be more concise to feature-flag the module that exports `OsslDtlsCert`? In other words, create a mock module that mimics the API of `OsslDtlsCert`? Then this module doesn't need to have any `cfg`s at all.

Reply:
But then, like this I can use the DummyCert for both wincrypto and openssl. So it's one reusable dummy component instead of two separate dummy impls.

Reply:
They need to have the same API anyway, right? You can re-export the same struct under different names to share the impl.